app/services/ExactTargetService.scala

```diff
@@ -226,8 +226,10 @@ class ExactTargetService(
 }
 
 object SqsClient extends LazyLogging {
-  private val sqsClient = new AmazonSQSClient(CredentialsProvider)
-  sqsClient.setRegion(Region.getRegion(Regions.EU_WEST_1))
+  private val sqsClient = AmazonSQSClient.builder
```

Comment:
Yeah, I haven't quite settled on the best style for this builder stuff yet... think I'll go with this for now.